ipam/pool.go

```diff
@@ -159,6 +159,9 @@ func (am *addressManager) setAddressSpace(as *addressSpace) error {
 	if !ok {
 		am.AddrSpaces[as.Id] = as
 	} else {
+		// merges the address set refreshed from the source
+		// and the ones we have currently in this address space
+		log.Printf("[ipam] merging address space")
```

Comment:
can you as add log before am.save() saying "saving ipam state"

Reply:
Done